### PR TITLE
Add performance job

### PR DESCRIPTION
**Pull request by cherrycl**
Signed-off-by: Cherry Wang <cherry@iotechsys.com>

1. Influxdb host, 3 test clients and jmeter node are defined by parameter.
2. InfluxDB host, first test client and jmeter node are required. Whether Second and third test client are enabled that depends on testcases quantities.

### CODE REVIEW

**Review by jamesrgregg — COMMENTED** [review]
Have you tested this in the sanbox yet?

File: jjb/performance-testing/performance-test.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+
+- project:
+    name: performance-testing
+    project-name: performance-testing
```
Comment (eb-oss) [inline jjb/performance-testing/performance-test.yaml:5]: There is currently a "performance-test" repo in edgexfoundry-holding, but it does not have a Jenkinsfile.

File: jjb/edgex-templates-performance.yaml [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+      - lf-infra-github-scm:
+          url: '{git-clone-url}{github-org}/{project}'
+          refspec: ''
+          branch: 'refs/heads/{branch}'
+          submodule-recursive: '{submodule-recursive}'
+          choosing-strategy: default
+          jenkins-ssh-credential: '{jenkins-ssh-credential}'
+          submodule-timeout: 10
+
+    #triggers:
```
Comment (eb-oss) [inline jjb/edgex-templates-performance.yaml:101]: Is the plan to remove all of this commented-out code once testing is complete, before merging? (this applies to the several sections below as well; I'll just put the comment here)

**Review by eb-oss — CHANGES_REQUESTED** [review]

File: jjb/performance-testing/performance-test.yaml (hunk 1 above)
Comment (cherrycl) [reply jjb/performance-testing/performance-test.yaml:5]: Sorry, I didn't point out the Jenkinsfile is on reviewing, please see branch [issue-7](https://github.com/edgexfoundry-holding/performance-test/tree/issue-7).

**Comment by cherrycl** [comment]
> Have you tested this in the sanbox yet?
@jamesrgregg  I didn't test in the sandbox. I don't have authorization. Does anybody help this?

File: jjb/edgex-templates-performance.yaml (hunk 2 above)
Comment (cherrycl) [reply jjb/edgex-templates-performance.yaml:101]: Yes, I will remove the commented code after testing is complete on sandbox. Thanks.

**Comment by jamesrgregg** [comment]
@cherrycl I’ll request sandbox access for you.

**Comment by jamesrgregg** [comment]
@cherrycl good job getting the job landed on the sandbox.  It looks like the job is failing because it needs docker-compose installed.  We want to see a successful test with data stored up to the InfluxDB before we accept this PR.

**Comment by jamesrgregg** [comment]
@cherrycl are you ready to get this moved out of edgexfoundry-holding?

**Comment by cherrycl** [comment]
@jamesrgregg Do you mean moving performance test repository from edgexfoundry-holding to edgexfoundry? I just changed the url of repository to edgexfoundry in jjb file. Once the performance test move to edgexfoundry github, it should be worked.

**Review by jamesrgregg — APPROVED** [review]
LGTM

**Review by eb-oss — CHANGES_REQUESTED** [review]
This needs to be properly rebased so that only the changes unique to this branch show up. Also, my feedback has not been addressed: references to `performance-testing` need to be changed to `performance-test`, and commented-out code needs to be removed.

**Pull request closed without merging**